plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationManagerImpl.kt

```diff
@@ -394,15 +396,15 @@ abstract class AnnotationManagerImpl<G : Geometry, T : Annotation<G>, S : Annota
   }
 
   private fun updateDragSource() {
-    delegateProvider.getStyle { style ->
+    style.get()?.let { style ->
```

Comment:
What if style of MapboxMap changes? Will the AnnotationManager use outdated style?

Reply:
There is no need to change annotation manager, I'm working on a new solution.